runtime/compiler/runtime/MetaData.cpp

```diff
@@ -1555,7 +1578,11 @@ createMethodMetaData(
 
    populateInlineCalls(comp, vm, data, callSiteCursor, numberOfMapBytes);
 
-   if (!(vm->_jitConfig->runtimeFlags & J9JIT_TOSS_CODE) && !vm->isAOT_DEPRECATED_DO_NOT_USE())
+   if (!(vm->_jitConfig->runtimeFlags & J9JIT_TOSS_CODE) && !vm->isAOT_DEPRECATED_DO_NOT_USE()
+#if defined(JITSERVER_SUPPORT)
+      && comp->getPersistentInfo()->getRemoteCompilationMode() != JITServer::SERVER
```

Comment:
Since we have the comp object, a better test could be comp->isOutProcessCompilation()